js/search/mediaObjectModel.js

```diff
@@ -43,7 +43,17 @@ const MediaObjectModel = new Lang.Class({
         'width': GObject.ParamSpec.uint('width',
             'Width', 'The width of the media in pixels',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
-            0, GLib.MAXUINT32, 0)
+            0, GLib.MAXUINT32, 0),
+
+        /**
+         * Property:poster-uri
+         * URI of the the video's poster image
```

Comment:
Does this need to be on `MediaObjectModel` as opposed to just on `VideoObjectModel`?